src/gitingest/schemas/ingestion.py

```diff
@@ -83,6 +83,8 @@ class IngestionQuery(BaseModel):  # pylint: disable=too-many-instance-attributes
         The patterns to include.
     include_submodules : bool
         Whether to include all Git submodules within the repository. (default: ``False``)
+    s3_url : str | None
+        The S3 URL where the digest is stored if S3 is enabled (default: ``None``).
```

Comment:
We leave the default specification out for `None` in the rest of the codebase. 
```suggestion
        The S3 URL where the digest is stored if S3 is enabled.
```